### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/03/day2.html?fbclid=IwAR3jTI_-UuIGBh8BpyFTu5TNJS1xuANSLGfSraMBD8tDCMHqR7vj6Y_agzs_aem_Af94nEiRVZzjeuNIBm6UitD_aQsdLugJs2uajbqMYRX0KG35PKaPL7O-WSh8ASiBPxQ

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#8] [Comments]
# https://mnfienen.github.io/talulat/2023/08/03/day2.html?fbclid=IwAR3jTI_-UuIGBh8BpyFTu5TNJS1xuANSLGfSraMBD8tDCMHqR7vj6Y_agzs_aem_Af94nEiRVZzjeuNIBm6UitD_aQsdLugJs2uajbqMYRX0KG35PKaPL7O-WSh8ASiBPxQ

Day 2 - An early rest day and an opportunity to hang out with friends and do some logistics miles today: 12 total miles: 76 km today: 20 total km: 142

https://mnfienen.github.io/talulat/2023/08/03/day2.html?fbclid=IwAR3jTI_-UuIGBh8BpyFTu5TNJS1xuANSLGfSraMBD8tDCMHqR7vj6Y_agzs_aem_Af94nEiRVZzjeuNIBm6UitD_aQsdLugJs2uajbqMYRX0KG35PKaPL7O-WSh8ASiBPxQ

**Comment by aleaf** [comment]
Ugh, sorry to hear about the wheel!

**Reply by mnfienen** [reply]
always gonna be something....
